### ISSUE TITLE
Add new (best) free DDNS provider

### ISSUE THREAD

**Issue by lockheed** [stephdl/nethserver-ddclient#3]
I think it would be beneficial to add now-dns.com
Of the all free ddns providers, this one is by far the best one I found. It it reliable, has good selection of decent domains, and - most importantly - does not require any periodic reconfirmations! It is set it and forget it!
I am surprised it is not listed.

Anyways, details of its API can be found here: https://now-dns.com/?m=api

**Comment by stephdl** [comment]
Hi @lockheed 

My concern is that you have plenty of providers, hard to follow all of them

When I look to the configuration
```
ssl=yes                                 # use ssl-support.  Works with
protocol=dyndns2			#
daemon=1m                               # check every 60 seconds
mail=root                               # mail all msgs to root
mail-failure=root                       # mail failed update msgs to root
pid=/var/run/ddclient.pid               # record PID in file.
use=web, web=now-dns.com/ip      	# get ip from server.
server=now-dns.com               	# default server
login=<email>            		# default login
password=<password>                     # default password
server=now-dns.com,              \
<your hostname>
```
It looks it should work out of the box with the custom field in the ddclient panel.
I would be pleased if you can do a PR to ddclient. take a look to the commit to see what to modify

https://github.com/stephdl/nethserver-ddclient/commit/a72ce8bd96c38a0638df507bd4eed21f549337d1

another example 

http://community.nethserver.org/t/need-tests-nethserver-ddclient/572/36?u=stephdl

**Comment by lockheed** [comment]
I managed to make this work with custom provider option. It was quit simple, as you wrote.
